### DISCUSSION TITLE
Cannot start wiki Service after Update to 2.5.285

### DISCUSSION THREAD

**Discussion by mx2fr** [requarks/wiki#5454] [Error / Bug Report]
Hi,

since I´ve updated to Version 2.5.285 the wiki.sbervice cannot start anymore.

Here are the Starting logs:

Jul 08 10:26:01 rzv-vwiki01 systemd[1]: Started Wiki.js.
Jul 08 10:26:01 rzv-vwiki01 node[2374]: Loading configuration from /var/www/wiki/config.yml... OK
Jul 08 10:26:01 rzv-vwiki01 node[2374]: 2022-07-08T08:26:01.866Z [MASTER] info: =======================================
Jul 08 10:26:01 rzv-vwiki01 node[2374]: 2022-07-08T08:26:01.868Z [MASTER] info: = Wiki.js 2.5.285 =====================
Jul 08 10:26:01 rzv-vwiki01 node[2374]: 2022-07-08T08:26:01.868Z [MASTER] info: =======================================
Jul 08 10:26:01 rzv-vwiki01 node[2374]: 2022-07-08T08:26:01.868Z [MASTER] info: Initializing...
Jul 08 10:26:02 rzv-vwiki01 node[2374]: 2022-07-08T08:26:02.224Z [MASTER] info: Using database driver pg for postgres [ OK ]
Jul 08 10:26:02 rzv-vwiki01 node[2374]: 2022-07-08T08:26:02.226Z [MASTER] info: Connecting to database...
Jul 08 10:26:02 rzv-vwiki01 node[2374]: 2022-07-08T08:26:02.238Z [MASTER] info: Database Connection Successful [ OK ]
Jul 08 10:26:02 rzv-vwiki01 node[2374]: 2022-07-08T08:26:02.615Z [MASTER] info: Loading GraphQL Schema...
Jul 08 10:26:02 rzv-vwiki01 node[2374]: 2022-07-08T08:26:02.990Z [MASTER] error: globalThis is not defined
Jul 08 10:26:02 rzv-vwiki01 systemd[1]: wiki.service: Main process exited, code=exited, status=1/FAILURE
Jul 08 10:26:02 rzv-vwiki01 systemd[1]: wiki.service: Failed with result 'exit-code'.
Jul 08 10:26:03 rzv-vwiki01 systemd[1]: wiki.service: Scheduled restart job, restart counter is at 5.
Jul 08 10:26:03 rzv-vwiki01 systemd[1]: Stopped Wiki.js.
Jul 08 10:26:03 rzv-vwiki01 systemd[1]: wiki.service: Start request repeated too quickly.
Jul 08 10:26:03 rzv-vwiki01 systemd[1]: wiki.service: Failed with result 'exit-code'.
Jul 08 10:26:03 rzv-vwiki01 systemd[1]: Failed to start Wiki.js.

Please Help me :(

**Comment by D4thToMS** [comment]
I have the same exact issue with the same version.
Installed wiki.js 2.5.285 on CentOS 8 with mysql DB, when launching through **node server** 
# node server
Loading configuration from /var/www/wiki.js/config.yml... OK
2022-08-02T05:13:03.335Z [MASTER] info: =======================================
2022-08-02T05:13:03.337Z [MASTER] info: = Wiki.js 2.5.285 =====================
2022-08-02T05:13:03.337Z [MASTER] info: =======================================
2022-08-02T05:13:03.338Z [MASTER] info: Initializing...
2022-08-02T05:13:03.908Z [MASTER] info: Using database driver mysql2 for mysql [ OK ]
2022-08-02T05:13:03.911Z [MASTER] info: Connecting to database...
2022-08-02T05:13:03.945Z [MASTER] info: Database Connection Successful [ OK ]
2022-08-02T05:13:04.385Z [MASTER] warn: Mail is not setup! Please set the configuration in the administration area!
2022-08-02T05:13:04.487Z [MASTER] info: Loading GraphQL Schema...
2022-08-02T05:13:05.041Z [MASTER] error: globalThis is not defined

So -> https://github.com/Requarks/wiki/discussions/4411 
After switching to node 14.X  (https://www.server-world.info/en/note?os=CentOS_Stream_8&p=nodejs&f=3) everything is fine
